cfgov/jinja2/v1/owning-a-home/explore-rates/index.html

```diff
@@ -307,48 +303,61 @@ <h3>Data table</h3>
                         </table>
                     </div>
                     <section id="chart-section" class="chart">
+                        <div class='chart-menu'>
+                            <button class="chart-menu_btn
+                                           cf-icon
+                                           cf-icon__after
+                                           cf-icon-down">
+                                Download chart
```

Comment:
After clicking on this I can't click it again to collapse it. I have to click off the menu. Not sure if that's existing behavior or not.

Reply:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/4047/commits/805de5edd2f3f85ba299ecef445c033fc1e117a9. I had to add JS to get his to work.